### PR TITLE
Add postgres user to `pg_isready` in CI

### PR DESCRIPTION
**Pull request by di**
Should resolve our `"FATAL: role "root" does not exist"` errors, at least.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
         env:
           POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
         # Set health checks to wait until postgres has started
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -U postgres -d postgres
```
Comment (miketheman) [inline .github/workflows/ci.yml:72]:
```suggestion
        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 --username postgres --dbname postgres
```

Comment (miketheman) [reply .github/workflows/ci.yml:72]: Also might be an args order problem - wrap all of `health-cmd` args in a single quoted string

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -69,7 +69,7 @@ jobs:
         env:
           POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
         # Set health checks to wait until postgres has started
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 --username postgres --dbname postgres
```
Comment (miketheman) [inline .github/workflows/ci.yml:72]:
```suggestion
        options: --health-cmd "pg_isready --username postgres --dbname postgres" --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment (di) [inline .github/workflows/ci.yml:72]:
```suggestion
        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 --username=postgres --dbname=postgres
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -69,7 +69,7 @@ jobs:
         env:
           POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
         # Set health checks to wait until postgres has started
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 --username=postgres --dbname=postgres
```
Comment (di) [inline .github/workflows/ci.yml:72]:
```suggestion
        options: --health-cmd "pg_isready --username=postgres --dbname=postgres"--health-interval 10s --health-timeout 5s --health-retries 5
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -69,7 +69,7 @@ jobs:
         env:
           POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
         # Set health checks to wait until postgres has started
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options: --health-cmd "pg_isready --username=postgres --dbname=postgres"--health-interval 10s --health-timeout 5s --health-retries 5
```
Comment (di) [inline .github/workflows/ci.yml:72]:
```suggestion
        options: --health-cmd "pg_isready --username=postgres --dbname=postgres" --health-interval 10s --health-timeout 5s --health-retries 5
```

**Comment by di** [comment]
Oooh, two whole test runs without hanging here, 🤞 this resolves that issue too.

**Pull request merged**